PodsTest/Podfile

```diff
@@ -0,0 +1,10 @@
+platform :ios, '13.0'
```

Comment:
Shouldn't it be iOS 11 if we support it?

Reply:
it is not important, it is an additional project to check the installation of pods locally (check subspec file)